topics/statistics/tutorials/clustering-machinelearning/tutorial.md

```diff
@@ -0,0 +1,482 @@
+>                - {% icon param-select %} *"Affinity"*: `Euclidean`
+>                - {% icon param-select %} *"Linkage"*: `ward`
+> 2. Rename the generated file to `circles hierarchical clustering`
+{: .hands_on}
+
+
+Then , you can visulaze the clustering result using the following steps:
+
+> ### {% icon hands_on %} Hands-on: Visualize hierarchical clutsreing result on circles data
+>
+> 1. **Scatterplot w ggplot2** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Input tabular dataset"*: **circles hierarchical clustering**
+>    - *"Column to plot on x-axis"*: `1`
+>    - *"Column to plot on y-axis"*: `2`
+>    - *"Plot title"*: `Hierarchical Clustering in Circles data`
+>    - *"Label for x axis"*: `Sepal length`
+>    - *"Label for y axis"*: `Sepal width`
+>    - In *"Advanced Options"*:
+>        - *"Data point options"*: `User defined point options`
+>            - *"relative size of points"*: `2.0`
+>        - *"Plotting multiple groups"*: `Plot multiple groups of data on one plot`
+>            - *"column differentiating the different groups"*: `3`
+>            - *"Color schemes to differentiate your groups"*: `Set 2 - predefined color pallete`
+>
+> 2. **View** {% icon galaxy-eye%} the resulting plot:
+
+
+In the next steps, you can apply these three algorithms (hierarchical, k-means and DBSCAN) in simlar steps to moon and circles datasets. For the DBSCAN algorithm, the parameters are not the default parameters and you should set them as follows: for the circles data set ( Maximum neighborhood distance=0.2 and Minimal core point density=5) and for the moon datasets (Maximum neighborhood distance=0.3 and Minimal core point density=4). You can see the scatter plots of the clustering results in Figure 13 and Figure 14. 
+
+>    ![data](images/circles_clustering.png "Plot of Clustering Algorithms on Circles Data")
+
+>    ![data](images/moon_clustering.png "Plot of Clustering Algorithms on Moon Data")
+
+
+# Conclusion
+
+In this tutorial, we discussed the unsupervised learning task: clustering. Clustering is used to discover structure in unlabeled data. You learned about the
+hierarcial , k-means and DBSCAN algorithms. By following these steps, we learned how to perform clustering algorithms using clustering and plotting tools in Galaxy. Moreover, we visualized the results using multiple plots to analyse the clustering tasks. There are many other clustering approaches which can be tried out on these datasets to find how they perform. Different datasets can also be analysed using these algorithms. The clustering algorithms have some parameters which can be altered while performing the analyses to see if they affect the clustering or not. Although clustering is easy to implement, you need to take care of some important aspects like treating outliers in your data and making sure each cluster has sufficient population. Some data pre-processors can also be used to clean the datasets.
```

Comment:
` we learned how to perform clustering and visualize results using clustering and plotting tools respectively in Galaxy`

Reply:
Can't say `the clustering is easy to implement`? Based on which evidence? Maybe some datasets are easy to cluster but not all

Reply:
Please rephrase conclusion.

Reply:
The comments are applied.